perms/perms.go

```diff
@@ -308,6 +309,16 @@ func MacaroonWhitelist(allowPublicUniProofCourier bool,
 		whitelist[k] = v
 	}
 
+	// Conditionally whitelist universe server read methods.
+	if allowUniPublicAccessRead {
```

Comment:
nit: to make things a bit more explicit, we could do `if allowUniPublicAccessRead || allowPublicUniProofCourier {` since the proof courier flag overwrites/supersedes the other two.